seabird.proto

```diff
@@ -166,6 +167,14 @@ message StreamInfoResponse {
 
   map<string, CommandMetadata> commands = 3;
 
-  // TODO: find a way to track IPs related to plugins - maybe deny connections
-  // from an IP other than the registration
+  string ip = 4;
+  uint64 connection_time = 5;
+}
+
+message CoreInfoRequest {
+}
+
+message CoreInfoResponse {
+  string nick = 1;
```

Comment:
In other places we've called this current_nick. I think this is fine because it has an implied "current" when you make the request, but what do you think?

Reply:
I agree with both, but I'd like to make it consistent with the rest of the file. Sorry about that!